assets/js/googlesitekit/datastore/user/user-input-settings.js

```diff
@@ -109,7 +109,7 @@ const baseActions = {
 		const registry = yield Data.commonActions.getRegistry();
 		yield clearError( 'saveUserInputSettings', [] );
 
-		const settings = registry.select( STORE_NAME ).getUserInputSettings();
+		const settings = registry.select( STORE_NAME ).getUserInputSettings() || {};
```

Comment:
It seems like maybe a test should be added/updated for the changes to this action.

Reply:
This seems to be not needed anymore. Reverted it.